common/tzif.ksy

```diff
@@ -0,0 +1,262 @@
+meta:
+  id: tzif
+  title: The Time Zone Information Format (TZif)
+  xref:
+    mime: application/tzif
+    # mime: "application/tzif-leap" if leap-second records are included
+    rfc: 8536
+  license: CC0-1.0
+  ks-version: 0.8
+  endian: be
+doc: |
+  TZif is the binary file format used to represent data for a single time zone
+  compiled by the _zic_ compiler from information in the IANA time zone
+  database. It is used by most UNIX systems to calculate local time for a time
+  zone, usually identified by a time zone identifier string referring to a
+  relatively populous location within the geographic region covered by the time
+  zone, for example: "America/New_York" or "Europe/Paris". This identifier is
+  external to the TZif file itself.
+doc-ref: https://tools.ietf.org/html/rfc8536
+seq:
+  - id: v1_struct
+    type: tz_struct(false)
+  - id: v2_struct
+    type: tz_struct(true)
+    if: version >= 2
+  - id: v2_footer
+    type: tz_footer
+    if: version >= 2
+instances:
+  version:
+    value: v1_struct.header.version
+types:
+  tz_struct:
+    params:
+      - id: is_ts64
+        type: bool
+        doc: |
+          Chooses whether to use 64-bit or 32-bit time values. Set to `false`
+          for 32-bit time values (used in the version 1 data block). Set to
+          `true` for 64-bit time values (used in the version 2+ data block).
+    seq:
+      - id: header
+        type: tz_header
+      - id: data
+        type: tz_data(is_ts64)
+  tz_header:
+    doc-ref: https://tools.ietf.org/html/rfc8536#section-3.1
+    seq:
+      - id: magic
+        contents: TZif
+      - id: ver_byte
+        -orig-id: ver
+        type: u1
+        doc: |
+          File format version: 0x00 Version 1; 0x32 Version 2; 0x33 Version 3.
+          The additional header block for version 2+ after the version 1 data
+          block MUST have the same version as as the first header block.
+      - id: reserved
+        size: 15
+        doc: Reserved bytes MUST be 0.
+      - id: num_is_ut_flags
+        -orig-id: isutcnt
+        type: u4
+        doc: |
+          Number of UT/local indicators contained in the data block. MUST
+          either be 0 or equal to `num_local_time_types`.
```

Comment:
`valid`

Reply:
https://github.com/kaitai-io/kaitai_struct/issues/435

Reply:
What should `ks-version` be set to if validation is used?

Reply:
To avoid forward references, I have validated `num_local_time_types` instead of `num_is_ut_flags`.